### PR TITLE
Basic v1 article

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
For when the app fails to get v2 data

### CODE REVIEW

**Comment by matthew-andrews** [comment]
i really didn't want to do this……

File: client/components/comments/main.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var fetchres = require('fetchres');
 
 module.exports = {};
 module.exports.init = function(uuid, flags) {
-	if (!flags.get('articleComments').isSwitchedOn) {
+	if (flags.get('articleComments').isSwitchedOff || !document.querySelector('#comments')) {
```
Comment (matthew-andrews) [inline client/components/comments/main.js]: isSwitchedOff is deprecated

**Comment by ironsidevsquincy** [comment]
understandable - kept it as simple as possible, just title, standfirst, byline and body. Think it's useful while v2 is still bedding in

**Comment by matthew-andrews** [comment]
then can we put it behind a flag that will go back to the old behaviour at the end of Q2?

**Comment by ironsidevsquincy** [comment]
sure can

**Comment by matthew-andrews** [comment]
perhaps reuse

**Comment by matthew-andrews** [comment]
articleCapiV1Fallback

File: client/components/comments/main.js (hunk 1 above)
Comment (commuterjoy) [inline client/components/comments/main.js]: Old habits die hard

https://github.com/guardian/frontend/blob/master/common/app/common/management.scala#L11

Comment (matthew-andrews) [inline client/components/comments/main.js]: https://github.com/Financial-Times/next-feature-flags-client/issues/26

**Pull request merged**